CMakeLists.txt

```diff
@@ -48,7 +48,7 @@ else()
     endif()
 endif()
 
-pkg_check_modules(LIBJPEG_TURBO libjpeg=2.1.1)
+pkg_check_modules(LIBJPEG_TURBO libjpeg>=2.1.1 libturbojpeg>=2.1.1)
```

Comment:
This will link libraries of both `libjpeg` and `libturbojpeg`. We should avoid linking additional libraries.
Can we do this instead? 
```
pkg_check_modules(LIBJPEG_TURBO libjpeg<=2.2)
```

Reply:
I have changed the check to a different one to remove the `libturbojpeg` linking